OxfordBodleian/Juel-Jensen/BDLaethg36.xml

```diff
@@ -98,13 +102,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                             <locus from="51r" to="54v"/>
                                             1, stub after 4
                                             3 added
-                                            <note><locus target="#54"/>  and <locus target="#52"/> probably formed an original bifolium, which has now become separated.
+                                            <note><locus target="#54"/> and <locus target="#52"/> probably formed an original bifolium, which has now become separated.
                                             <locus target="#53"/> is sewn to <locus target="#54"/>.</note>
+                                            <!-- EDS 01.12.2022: Could the stubs after f. 54 be cut leaves?-->
```

Comment:
There seems to be text visible on the last one, the only one visible in the images.
I actually would not know how to encode a cut leaf vs a stub formally in the quire structure...but a note could be made at the least.
What do the others think?

Reply:
Definetly let's not invent a new encoding system, could include a note if you believe it's a worthwhile feature!

Reply:
It looks like there are letters on the stubs, doesn't it? I am wondering if a text has been cut out...